ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/bfd_multihop.schema.yml

```diff
@@ -0,0 +1,27 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  bfd_multihop:
+    documentation_options:
+      filename: Fabric Variables
+      table: BFD Settings
+    type: dict
+    description: BFD Multihop tuning
+    keys:
+      interval:
```

Comment:
Should we need to add descriptions to all of these keys?

Reply:
All descriptions will be reviewed in the next rounds of PRs where we focus on building the documentation.